modules/acquia_cms_headless/tests/src/Functional/DashboardNextjsSitesTest.php

```diff
@@ -214,6 +214,7 @@ private function testClone(mixed $nextjsSitesFieldset): void {
     /** @var \Drupal\FunctionalJavascriptTests\JSWebAssert $assertSession */
     $assertSession = $this->assertSession();
     $nextjsSitesFieldset->findButton('List additional actions')->click();
+    $assertSession->waitForElementVisible('css', 'form.acquia-cms-headless-next-sites .dropbutton-wrapper .dropbutton .clone');
```

Comment:
@apathak18 i think you should use the form ID instead of form.acquia-cms-headless-next-sites

Reply:
added after a discussion